docs/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,28 @@
+# Troubleshooting
+If you're reading this, something went wrong.
+Don't worry though, as this is the most thorough guide to help you ~~(and to help us)~~!
+
+## `./Impostor.Server: line 1: ELF: not found` (plus other errors)
+No idea where you got that system. But we clearly do **NOT** support it.
+
+## `cannot execute binary file: Exec format error`
+Please check that you have downloaded the right version of Impostor, as we mantain two CPU architectures (x64 and ARM).  
+Unless you are running Impostor on a SBC (Single-Board Computer), like the Raspberry Pi, you most likely want to use the x64 version.
+
+## `./Impostor.Server: Permission denied`
+This is an error related to Linux file permissions.  
+You can solve this by doing: `chmod +x Impostor.Server`
+_(Note: only run `chmod` when you trust the software. Linux blocks downloaded executables by default for a reason)_
+
+## `You are using an older version of the game`
+You are using an older version of Impostor. The game does not really check who is outdated and blames it on the user.  
+Make sure you got the latest working version of Impostor (probably in AppVeyor, not Github).
+
+## `You disconnected from the server. Reliable Packet 1 ...`
+Please double-check that you have followed the [Server Configuration](Server-configuration.md) correctly.  
+**NOTE: Your public ip does not start with `127` nor `192`**  
```

Comment:
or <https://en.wikipedia.org/wiki/Reserved_IP_addresses>.

Reply:
Could be, but I summed it up because those 2 cases are the most common ones